src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -113,8 +103,12 @@ export default function VideoVisitLocation({ header, appointment, facility }) {
             className="far fa-calendar vads-u-margin-right--1"
           />
           <AddToCalendar
-            summary={`${header}`}
-            description={`instructionText`}
+            summary={summary}
+            description={{
+              text,
+              phone: formatFacilityPhone(facility),
```

Comment:
Could this phone item go in the `selectCalendar` data function? I wonder if it makes sense to return the `description` object from `selectCalendarData`?

Alternatively, do we need to group these properties into a `description` object at all?

Reply:
I guess they don't have to but I don't know where they would go. Individual functions maybe?

Reply:
I was just thinking about different ways of making the output from `selectCalendarData` and the props on `AddToCalendar` work together more cleanly. We create this description object every time we use AddToCalendar, but it doesn't look like we gain much by doing that over using individual `text`, `phone`, and `additionalText` props.

Alternatively, if we do think having those three pieces of data grouped into a `description` object is helpful, it would potentially be cleaner if `selectCalendarData` returned a `description` object, instead of those pieces of data individually.

Reply:
Whatever we decide here, I think the work to update it can be deferred to a separate ticket. Since it doesn't affect the functionality of these changes, I don't want to hold up this PR for it.